### ISSUE TITLE
NTR: Pharyngeal biopsy: myxomas (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9370]
New term request Pharyngeal biopsy: myxomas (elthat_150214030439): Oropharyngeal, pharynx

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000074
- label: Pharyngeal myxoma
- parents: [Neoplasm of the pharynx (HP:0100638)](https://hpo.jax.org/app/browse/term/HP:0100638)
- pmid: PMID:23984217
- def: A myxoma (benign, locally infiltrative neoplasms arising from mesenchymal origin) localized to the pharynx.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9370
- synonyms:
